.travis.yml

```diff
@@ -21,7 +21,6 @@ env:
     - MAGENTO_VERSION="magento-mirror-1.8.1.0" INSTALL_SAMPLE_DATA=yes
     - MAGENTO_VERSION="magento-mirror-1.9.2.1" INSTALL_SAMPLE_DATA=no
     - MAGENTO_VERSION="magento-mirror-1.9.2.4" INSTALL_SAMPLE_DATA=no
-    - MAGENTO_VERSION="magento-mirror-1.9.3.0" INSTALL_SAMPLE_DATA=no
 
```

Comment:
This version isn't available in n98-magerun version 1.97.22 which causes this test to fail. my workaround solution is using byte's latest mirror. I hope to add it again when i revised all tests. otherwise, we could use the other bash file that is used to download byte's mirror tar but then use the zip from the openmage mirror just like i did for the magento-malware-scanner [https://github.com/gwillem/magento-malware-scanner/pull/17](PR with setup that downloads Magento from openmage mirror)